package.json

```diff
@@ -42,8 +42,12 @@
     ]
   },
   "devDependencies": {
+<<<<<<< HEAD
     "execa": "^5.1.1",
     "mr-pdf": "^1.0.7",
     "wait-on": "^6.0.0"
+=======
+    "docusaurus": "^1.14.7"
+>>>>>>> issue-44
   }
```

Comment:
I think you forgot to erase the `<<<<<<<<HEAD`, `======`, `>>>>>>issue-44`tag when you merge it to the main branch on your repository.

Reply:
Good point that could be why deployment failed. Nice spot :D